astro/src/content/json/scopes/update-response-body.json

```diff
@@ -0,0 +1,13 @@
+{
+  "scope": {
+    "applicationId": "8c9f82c8-0926-4c6c-b8be-1bcface5b378",
+    "data": {},
+    "defaultConsentDetail": "This will provide the requesting application read-only access to your data",
+    "defaultConsentMessage": "View your data",
+    "id": "f32b777a-8cc4-4233-afcf-d4c42d3ca488",
+    "insertInstant": 1712268302394,
+    "lastUpdateInstant": 1712268302394,
+    "name": "data:read",
+    "required": false
```

Comment:
We didn't explicitly update it, but it would get set back to the default value of `false` if not on the `PUT` request. To make this more clear, I can treat this example as a `PATCH` or just be more explicit in what is being updated.

Reply:
I think I was changing this so that the update JSON would be different than the create JSON. (Otherwise why update?)

But maybe you don't need to change the `required` attribute.